### ISSUE TITLE
Typing of Eliom_content.Html5.*.a should prevent the use of a_href

### ISSUE THREAD

**Issue by pveber** [ocsigen/eliom#191]
The current type of `Eliom_content.Html5.F.a` permits the following expression
```ocaml
a ~service ~a:[a_href "#foo"] [ pcdata "foo" ] ()
```
which generates two `href` attributes in the `a` element.

**Comment by vasilisp** [comment]
Good point, it should be fairly easy to prevent this.

**Comment by pveber** [comment]
Yes, it means that there should be a `Html5_types.a_attrib` and a `Html5_types.raw_a_attrib`.
